.github/workflows/sync-version-with-upstream.yml

```
@@ -22,7 +22,15 @@ jobs:
         with:
           token: ${{ secrets.SNAPCRAFTERS_BOT_COMMIT }}
           update-script: |
+            #!/bin/bash
             # Update the snapcraft.yaml to match upstream version
             API="https://api.github.com/repos/jenkinsci/jenkins/releases/latest"
-            VERSION=$(curl -s "${API}" | grep -i name | awk 'FNR == 3 {print $2}' | cut -d ',' -f1)
-            sed -i 's/^\(version: \).*$/\1'"$VERSION"'/' snap/snapcraft.yaml
+            VERSION=$(curl -s "${API}" | grep -i name | awk 'FNR == 3 {print $2}' | cut -d ',' -f1 | sed 's/"//g')
```

```suggestion
            VERSION="$(curl -s "${API}" | jq -r '.name')"
```

Adds quotes, and I think is a bit more readable? :)

---

I wasn't aware that `jq` could be leveraged so I stuck with whatever tools would typically found in a basic Linux build.

---

`jq` is present on all Github Actions runners by default, so you're good to use it liberally :)